docs/cli/ilab-local-config.md

```diff
@@ -0,0 +1,184 @@
+
+
+### Manging model data
+
+
+New model checkpoints would be generated during training and
+saved to the `checkpoints/` directory along with a unique ref. 
+When base models are downloaded, they would be saved in the `base_models` directory with their basename being the unique ref.
+
+
+An `ilab model list` command would be added which allows the user to list out existing models. This would allow users to view all models, base models, trained checkpoints, etc. Additionally, the user could further filter out trained checkpoints based on which training phase they were generated in, i.e. `phase00`, `phase05`, and `phase10`.
+
+We can iterate on the behavior and optimize for UX, but the
+idea is you should be able to run either: `ilab model list` or
+`ilab list models` which produces a full list of all models on the local system.
+
+The output of `ilab model list` would look like the following:
+
+```
+MODEL ID    BASE MODEL                    SIZE    DATA TYPE    DATASET TRAINED WITH
+abcde       ibm-granite/granite-7b-base   28Gi    FP32         phase00-dataset-id
+cbdea       abcde                         28Gi    FP32         phase05-dataset-id
+```
+
+
+
+Finally, we would add a new `ilab model delete <ref>`  command 
+which would enable us to pass a reference to the model we want
+to delete.
+
+
+### Managing datasets
+
+New datasets would be generated through `ilab generate` (or whatever the current equivalent is) and the corresponding dataset would be saved under the `datasets/` directory. Each model would also have a unique ref associated with it.
+
+Following this, we could list out the datasets with `ilab data list` and further filter them out with either `knowledge` and `skills`, or `phase00`, `phase05`, and `phase10` depending on which SDG scheme is being used.
+
+Here's an example output from `ilab data list`
+
+```
+DATASET ID            PHASE TYPE        SIZE    NUM SAMPLES    TEACHER MODEL
+phase00-dataset-id    knowledge_base    500MB   650k           mistralai/mixtral-8x7B
+phase05-dataset-id    knowledge         230MB   330k           mistralai/mixtral-8x7B
+phase10-dataset-id    skils             500MB   550k           mistralai/mixtral-8x7B
+```
+
+An `ilab data delete <ref>` command would be added as well
+so that the user can easily delete generated datasets.
+
+Additionally, The `ilab model train` command would be able
+to receive a ref to a dataset for easy access.
+
+
+### Managing Eval Runs
+
+When we run evaluations, we expect to receive a certain value which is tied to a model checkpoint.
+
+Under this new framework, the evaluation outputs could be saved under the `eval_data` subdirectory, where `ilab` could later use to get the score for a particular model.
```

Comment:
Just as a note for reference, I have implemented this somewhat in https://github.com/instructlab/instructlab/pull/1369